pkgs/tools/networking/mailutils/default.nix

```diff
@@ -30,17 +30,15 @@
   guileSupport ? true,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "mailutils";
   version = "3.19";
 
   src = fetchurl {
-    url = "mirror://gnu/${pname}/${pname}-${version}.tar.xz";
+    url = "mirror://gnu/mailutils/mailutils-${finalAttrs.version}.tar.xz";
     hash = "sha256-UCMNIANsW4rYyWsNmWF38fEz+6THx+O0YtOe6zCEn0U=";
   };
 
-  separateDebugInfo = true;
-
```

Comment:
Why move this? Usually in my experience we put outputs (including separateDebugInfo) immediately after src.

Reply:
Undone.